src/components/calculator/SavedDataSelector.tsx

```diff
@@ -48,18 +65,35 @@ export const SavedDataSelector: React.FunctionComponent<{
         <Typeahead
           clearButton={true}
           emptyLabel={t('calculator.no_prebuilt_scenario_found')}
-          multiple
+          filterBy={filterByCallback}
           highlightOnlyResult={true}
           id="predefined-typeahead"
-          inputProps={{ autoComplete: 'chrome-off' }}
+          inputProps={{
+            autoComplete: 'chrome-off',
+            shouldSelectHint: (shouldSelect: boolean, event) => {
+              if (
+                event.keyCode === 13 /* return */ ||
+                event.keyCode === 9 /* tab */
```

Comment:
Is this working as intended? When I highlight an option and press tab, the dropdown closes and reverts to the previous selection.

Reply:
the event.keyCode === 13 part makes this work:

1. tab or click to focus input box
2. start typing indoor
3. the first entry that matches shows up as a hint
4. hitting enter selects the hint

It does seem like the keyCode === 9 part doesn't actually do anything. Will remove.

> When I highlight an option and press tab, the dropdown closes and reverts to the previous selection.

The sequence you describe doesn't work on the google search bar either and is also already present in the typeaheads on the current public-facing microcovid website.

Reply:
I have this fixed locally, but it requires another fix to the type definitions.